### PR TITLE
Fixed grid issue #3

### PR DESCRIPTION
**Pull request by harshitkathuria**
Fixed the shortcut links layout by making them a grid display

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -59,7 +59,12 @@ h1
 }
 
 /* Shortcut links */
+
+.icons {
+    width: 200px;
+}
+
```
Comment (sudo-rgorai) [inline styles.css]: I think there is no selector with a class named `icons`. You should change width property of `.icon` and not `.icons`. Currently it is specified in terms of `vw`. You can change it to `px`. When you do that, make sure you use a width such that each row has 5 icons by default on a standard screen. Thanks again for the help :smiley:

Comment (harshitkathuria) [reply styles.css]: Sorry..it was a bad error.😅. But I tried different workarounds and implemented what seemed the best. Please check if it is ok. Thanks

**Comment by sudo-rgorai** [comment]
LGTM :+1:

**Comment by harshitkathuria** [comment]
Thanks :)

**Pull request merged**